frontend/src/app/services/net.metrics/net.metrics.service.ts

```diff
@@ -236,17 +220,52 @@ export class NetMetricsService {
       openedChannels: openedChannels.length,
       closedChannels: closedChannels.length,
       settledChannels: settledChannels.length,
-      channelsPerParticipant: channelsPerParticipant,
-      participants: uniqueParticipants.length,
+      channelsPerAccount: channelsPerAccount,
+      participants: uniqueAccounts.length,
       topChannelsByDeposit: topChannelsByDeposit,
       averageDepositPerChannel: channelAverage || 0,
       averageDepositPerParticipant: averagePerParticipant || 0,
-      uniqueParticipants: uniqueParticipants,
+      uniqueAccounts: uniqueAccounts,
       totalNetworkDeposits: TokenUtils.toDecimal(totalNetworkDeposits, decimals)
     };
   }
 
-  //noinspection JSMethodCanBeStatic
+  // noinspection JSMethodCanBeStatic
+  private calculateChannelsPerAccount(accountsWithOpenChannels: UserAccount[]) {
+    const openChannels = accountsWithOpenChannels.reduce((acc, userAccount) => acc + userAccount.channels.length, 0);
+    return openChannels / accountsWithOpenChannels.length;
```

Comment:
What happens here if `accountsWithOpenChannels.length` is 0, so this is division by 0?

Reply:
Yes I completely didn't think about it. if `accountsWithOpenChannels` is empty then we have a 0 by 0 division which returns a `NaN` I will go fix it thanks